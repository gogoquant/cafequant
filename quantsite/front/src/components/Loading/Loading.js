import React from 'react';
import { Spin } from 'antd';
import styles from './Loading.scss';

const Loading = () => (
  <div className={styles.loading}>
    <Spin size="large" />
  </div>
);

export default Loading;
