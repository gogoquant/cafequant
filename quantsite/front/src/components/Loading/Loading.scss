.loading {
  width: 100%;
  height: 100%;
  margin: auto;
  padding-top: 50px;
  text-align: center;
}
