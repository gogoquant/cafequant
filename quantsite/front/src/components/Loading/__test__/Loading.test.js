import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Loading from '../Loading';

describe('<Loading />', () => {
  it('should render correctly', () => {
    const tree = mount(<Loading />);
    expect(toJson(tree, { noKey: true, mode: 'deep' })).toMatchSnapshot();
  });
});
