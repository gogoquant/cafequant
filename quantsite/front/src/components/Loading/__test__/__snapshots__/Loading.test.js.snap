// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> should render correctly 1`] = `
<div>
  <div
    className="ant-spin ant-spin-lg ant-spin-spinning"
  >
    <span
      className="ant-spin-dot ant-spin-dot-spin"
    >
      <i
        className="ant-spin-dot-item"
      />
      <i
        className="ant-spin-dot-item"
      />
      <i
        className="ant-spin-dot-item"
      />
      <i
        className="ant-spin-dot-item"
      />
    </span>
  </div>
</div>
`;
